File: backend-nodejs/src/Models/DTO/OrderDTO.js
class OrderDTO{

    constructor(
        orderId,
        eventId,
        orderedAt,
        ticketCategoryId,
        numberOfTickets,
        totalPrice
    ) {
        this.orderId = orderId;
        this.eventId = eventId;
        this.orderedAt = orderedAt;
        this.ticketCategoryId = ticketCategoryId;
        this.numberOfTickets = numberOfTickets;
        this.totalPrice = totalPrice;
    }

}

module.exports = OrderDTO;

File: backend-dotnet/EventTicketSystem/EventTicketSystem/Models/DTOs/OrderDTO.cs
namespace EventTicketSystem.Models.DTOs
{
    public class OrderDTO
    {
        public long OrderId { get; set; }

        public long EventId { get; set; }
        
        public DateTime OrderedAt { get; set; }

        public long TicketCategoryId { get; set; }

        public int NumberOfTickets { get; set; }

        public decimal TotalPrice { get; set; }

    }
}
